experimental/framework/frs-client/src/interfaces.ts

```diff
@@ -42,13 +42,14 @@ export interface FrsContainerServices {
  * Since Frs provides user names for all of its members, we extend the IMember interface to include
  * this service-specific value. It will be returned for all audience members connected to Frs.
  */
-export interface FrsMember extends IMember {
+export interface FrsMember<T> extends IMember {
```

Comment:
```suggestion
export interface FrsMember<T = any> extends IMember {
```

Reply:
So that users are not forced to pass a type if they don't have any additionalDetails